common/components/CheckboxItem/CheckboxItem.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "common/abstracts" as *;
+
+.container {
+  display: flex;
+  border: 2px solid $grey;
+  border-radius: 8px;
+  padding: 0.75em 1.5em;
```

Comment:
Could perhaps use 1rem as font-size here, 1.125rem / 18px is the default size, but looks a bit big in the dialog. Since the padding etc uses em they will also become smaller, and it seem to look better. What do you think?

Reply:
agree. updated